### PR TITLE
Feature/postgres update

### PR DESCRIPTION
**Pull request by denis-yuen**
Looks like postgres stable version is already 13
Some modest performance improvements noted at https://aws.amazon.com/blogs/database/upgrading-from-amazon-rds-for-postgresql-version-9-5/

Didn't try a straight upgrade locally, but a dump, load, and migrate seems fine. 

snyk seems to be misbehaving

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]
We'll need to update RDS too.

**Review by garyluu — COMMENTED** [review]
Missing the docker-compose file changes

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java [hunk 1]
```diff
@@ -175,7 +176,7 @@ public void testProprietaryAPI() {
      * @param path         Path of endpoint
      */
     private void testXTotalCount(Client jerseyClient, String path) {
-        Response response = jerseyClient.target(path).request().get();
+        Response response = jerseyClient.target(path).request().property(ClientProperties.READ_TIMEOUT,    0).get();
```
Comment (garyluu) [inline dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java:179]: Odd formatting

**Review by garyluu — APPROVED** [review]

**Review by Richard-Hansen — APPROVED** [review]
What were the Snyk issues? Dashboard seems OK on a cursory look.

**Comment by denis-yuen** [comment]
> What were the Snyk issues? Dashboard seems OK on a cursory look.

For commit `'test seems to require more time` snyk reported this set or test results which was odd https://app.snyk.io/org/denis-yuen/pr-checks/6f43e964-e48f-4908-9e57-c2ce138c2722?fromGitHubAuth=true

**Comment by Richard-Hansen** [comment]
> > We'll need to update RDS too.
> 
> Added [dockstore/dockstore-deploy#276](https://github.com/dockstore/dockstore-deploy/pull/276) there might not be much to do outside of the UI configuration

Hmm, that is odd. Hopefully just a one-off, but if we see it again, probably worth opening a ticket.

**Pull request merged**